soc/ti/k3/am6x/Kconfig

```diff
@@ -33,6 +33,7 @@ config SOC_SERIES_AM6X_R5
 	select SOC_EARLY_INIT_HOOK
 
 config SOC_PART_NUMBER
+	default "AM6232" if SOC_AM6232_M4
```

Comment:
The M4 functionality is the same on AM6254 and AM6232. Unclear what direction TI would like to see here.

cc @glneo